mesop/examples/testing/navigate_new_tab.py

```diff
@@ -0,0 +1,50 @@
+import mesop as me
+
+
+@me.page(path="/testing/navigate_new_tab")
+def page():
+  me.text("Navigate to New Tab Example", type="headline-5")
+  me.text("Click the buttons below to test navigation in new tabs:")
+  
+  with me.box(style=me.Style(margin=me.Margin.all(15))):
+    me.button("Open about page in new tab", on_click=navigate_to_about_new_tab)
```

Comment:
When I try to test this, a new tab opens to the right page. However the current page also changes to the same page as the new tab.